src/components/ChosenOne.css

```diff
@@ -0,0 +1,41 @@
+.chosen-one {
+  /* background-color: #f9f9f9;
+  background-size: cover; */
+  display: flex;
+  flex-direction: column;
+  z-index: 2;
+}
+
+.video {
+  content: ' ';
+  display: block;
+  position: absolute;
+  width: 100vw;
+  height: 100vh;
+  top: 0px;
+  left: 0;
+  padding-top: 800px;
```

Comment:
I wonder if this set padding might have to do with the difference in how the videos are displaying? Maybe having a margin instead would give more consistency between elements?